powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/logging/ObjectStepLogger.java

```diff
@@ -255,12 +255,26 @@ public void writeDoneFailed(String id) {
         return items;
     }
 
+    /**
+     * Get first {@link StepItem} with given ID.
+     * @param id Item ID.
+     * @return First {@link StepItem} with given id or {@code null} if logger doesn't contain such item.
+     */
+    public StepItem getFirstItem(String id) {
+        for (StepItem item: items) {
+            if (item.getId().equals(id)) {
```

Comment:
Are we should a null pointer exception is not possible in this condition?

Reply:
I took this code from the previously available methods that were checking by name and gave this some thought and yes, we are sure there cannot be NPE.

The `StepItem` and `StepError` do not have setters, only the constructor with all required parameters. We are the only ones creating these instances and we always assign an ID and name. We can add a null check, it won't solve much...

Reply:
I inverted the conditions and added null checks.